understanding/20/focus-order.html

```diff
@@ -62,30 +62,54 @@ <h2>Intent of Focus Order</h2>
       </p>
       
+      <p>Focus order needs to allow the user to navigate focusable elements in a logical order,
+         and that order needs to preserve any meaning or operation that the page is conveying. 
+         Focus order does not necessarily need to follow the visual layout of the web page,
+         as long as the order in which elements receive focus is logical, and the hierarchy and relationship
+         of content implied by the visual presentation is preserved. For instance, in a two-column
+         layout, the assumption in western left-to-right systems may be that focus moves through the elements in the
+         left-hand column first, from the top to the bottom of the column, and then proceed to the right-hand column.
+         However, if the two columns are independent of each other, and meaning/operation are not affected, it is not
+         a failure if elements in the right-hand column receive focus first, followed by the elements in the left-hand column.
+         As a best practice, though, make sure the focus order reinforces the reading order implied by the visual layout.
+      </p>
+      
+      <div class="note">
+         <p>It is not a failure of this Success Criterion if static/non-interactive content
+         receives focus. Making an element that does not act as an actionable user interface component
+         focusable, or programmatically moving focus to such an element, does not necessarily constitute
+         a failure of this Success Criterion. This situation is a failure if these additional focus stops
+         impede the operation of the content, or create confusing and/or illogical focus orders - for example,
+         a control appearing to receive focus multiple times due to the use of nested focusable elements.</p>
+         <pre><code>&lt;div tabindex="0"&gt;&lt;button&gt;...&lt;/button&gt;&lt;/div&gt;</code></pre>
+         <p>As a best practice, avoid having unwarranted focusable elements which cannot be operated
+         or actioned, as these are likely to make operation tedious for keyboard users.</p>
+      </div>
+      
       <p>For clarity:</p>
       
       <ul>
          
          <li>Focusable components need to receive focus in an order that preserves meaning and
-            operability only when navigation sequences affect meaning and operability.
+            operability.
          </li>
          
-         <li>In those cases where it is required, there may be more than one order that will preserve
-            meaning and operability.
-         </li>
+         <li>Focus order does not necessarily need to follow the visual presentation or layout.</li>
          
          <li>If there is more than one order that preserves meaning and operability, only one of
             them needs to be provided.
          </li>
          
+         <li>Static/non-interactive elements can receive focus, as long as they don't significantly
```

Comment:
We believe removing "significantly" is prudent. "Impede" already has a concept of hindering.
```suggestion
         <li>Static/non-interactive elements can receive focus, as long as they don't
```